### ISSUE TITLE
Mouse Block Select - Sublime keymap

### ISSUE THREAD

**Issue by porsager** [zed-industries/community#2067]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

Trying to do block select på holding down cmd+option and dragging / pressing with the mouse doesn't work.

It works with option+shift, but I can't find a way to remap to cmd+option, even though I would have suspected cmd+option to work when choosing the sublime keymap.

### Environment

Zed: v0.105.1 (preview)
OS: macOS 13.5.1
Memory: 32 GiB
Architecture: aarch64

### If applicable, attach your `~/Library/Logs/Zed/Zed.log` file to this issue.
If you only need the most recent lines, you can run the `zed: open log` command palette action to see the last 1000.

_No response_

**Comment by JosephTLyons** [comment]
I'm not entirely sure, but I think this is a general issue with Zed - it seems that we don't have a way to override the key binding for column select.  

Also, I read the [docs on Sublime's column select](https://www.sublimetext.com/docs/column_selection.html) and it would be great if we supported `subtract from selection` as well.

**Comment by porsager** [comment]
Yeah, I think my issue is actually wrong as what I was actually missing was the option+cmd solution to "add" more selections. Having both add and subtract would be great.

I'll just edit the title and issue to more clearly state this.
